packages/aft/lib/src/commands/publish_command.dart

```diff
@@ -0,0 +1,284 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:aft/aft.dart';
+import 'package:aft/src/util.dart';
+import 'package:graphs/graphs.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:pubspec_parse/pubspec_parse.dart';
+
+/// Command to publish all Dart/Flutter packages in the repo.
+class PublishCommand extends AmplifyCommand {
+  PublishCommand() {
+    argParser
+      ..addFlag(
+        'force',
```

Comment:
What type of error are you thinking we should be able to ignore?

Reply:
The `--force` flag is meant to be used in CI environments where prompts should be bypassed.